doc/1/controllers/index/m-delete/snippets/mDelete.cs

```diff
@@ -0,0 +1,3 @@
+var indexes = new JArray { "nyc-open-data", "mtp-open-data" };
```

Comment:
```suggestion
JArray indexes = JArray.Parse("['nyc-open-data', 'mtp-open-data']");
```

Reply:
It's better to be explicit with the types and also we use the `JArray.Parse` notation everywhere else because it's more readable with complex objects so it's better to stay consistent